### PR TITLE
Add Twitter SSO setup instructions

### PR DESCRIPTION
**Pull request by josephdpurcell**
Related to #110 but for Twitter

* Twitter SSO docs preview: https://github.com/josephdpurcell/nextcloud-social-login/blob/feature/twitter-sso-docs/docs/sso/twitter.md

### CODE REVIEW

File: docs/sso/twitter.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        * Organization website URL: https://example.com/
+    4. Click "Save"
+    5. Go to the "Keys and tokens tab"; here you should see a section "Consumer API keys", store these somewhere safe
+4. Configure "Social Login" in NextCloud
+    1. Click "Settings" in the menu
+    2. Under Administration click "Social login"
+    3. In the bottom section under "Twitter" enter:
+        * App id: the "API key" provided earlier
+        * Secret: the "API secret key" provided earlier
+        * Default group: "admin" or whatever user group you want new users assigned to
```
Comment (zorn-v) [inline docs/sso/twitter.md]: This is dangerous tip :wink:

File: docs/sso/twitter.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        * Allow login only from specified domain: (you can leave this blank)
```
Comment (zorn-v) [inline docs/sso/twitter.md]: It only applicable for google

File: docs/sso/twitter.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    4. Submit your application.
+    5. Confirm your email.
+    6. Wait for your application to be approved
+3. Create an application
+    1. Go to https://developer.twitter.com/
+    2. Click "Create an app"
+    3. Fill out relevant information:
+        * Website URL: https://example.com/
+        * Enable Sign In with Twitter: Check this box
+        * Callback URLs: https://example.com/apps/sociallogin/oauth/twitter
```
Comment (zorn-v) [inline docs/sso/twitter.md]: On some server configurations url may be https://example.com/index.php/apps/sociallogin/oauth/twitter

File: docs/sso/twitter.md (hunk 2 above)
Comment (josephdpurcell) [reply docs/sso/twitter.md]: Doh! Ok, fixing now on other PRs too

File: docs/sso/twitter.md (hunk 1 above)
Comment (josephdpurcell) [reply docs/sso/twitter.md]: Yes! I'll fix this and other PRs

File: docs/sso/twitter.md (hunk 3 above)
Comment (josephdpurcell) [reply docs/sso/twitter.md]: I'll add a generic statement explaining these instructions assume "https://example.com/" is the base URL.

**Comment by josephdpurcell** [comment]
I've made changes according to the comments -- it may not be significantly better, but hopefully it is sufficient.

**Pull request merged**